reliability/retry/retry_test.go

```diff
@@ -84,29 +87,32 @@ func Test_Retry_Execute(t *testing.T) {
 			expectErr:          true,
 		},
 	}
-	for name, tC := range testCases {
+	for name, tt := range testCases {
```

Comment:
Should we rename `testCases` to `tests`? To follow the convention we use in all other table test vars?

Reply:
done